### DISCUSSION TITLE
Possible to access forecast data in templates?

### DISCUSSION THREAD

**Discussion by andrewmooreio** [nathanmarlor/foxess_em#297] [Q&A] [answered]
Hi all,

Quick question. Is it possible to access the Solcast forecast within templates? I can't find any entities relating to it.

For this use case, I'm looking to have a card displaying when the peak solar output will be today.

The native `forecast.solar` integration has this functionality. However, because it uses a different prediction service, the values don't line up with the Solcast forecast. e.g. today Solcast forecasts the peak time at around 14:30 (based on the graph in the Energy dashboard) whereas the `forecast.solar` value is 13:00.

Thanks!

**Comment by nathanmarlor** [comment, marked as answer]
@andrewmooreio the raw forecast data is stored in the attributes of entity 'FoxESS EM: Forecast'.

![image](https://github.com/nathanmarlor/foxess_em/assets/3436911/8c581fcd-707f-4763-b447-c3b5500292dc)

**Reply by andrewmooreio** [reply]
Thanks @nathanmarlor, how did I not find that. Looks like the YAML will be challenging to parse in a template, but I'll give it a go!

**Reply by Wiggy93** [reply]
@andrewmooreio did you manage to parse the solcast data into a template?
